src/lib/openapi/spec/create-feature-schema.ts

```diff
@@ -32,14 +33,17 @@ export const createFeatureSchema = {
         },
         tags: {
             type: 'array',
+            description: 'Tags to add to the feature.',
             items: {
-                type: 'string',
-                example: 'simple:test',
+                $ref: '#/components/schemas/tagSchema',
```

Comment:
Fixing the schema to reuse existing schema and make it similar across unleash.

Reply:
Oh, wow, did we actually have the possibility to do this before?

Reply:
Wait, does this break the API somehow? As in, is this a breaking change?

Reply:
Nope, I implemented it yesterday, but after adding backend code, it made sense to use tagSchema.